.github/renovate.json

```
@@ -84,7 +60,7 @@
         "digest"
       ],
       "baseBranches": [
-        "/^release-1\\..*/",
+        "/^release-1\\.3/",
```

Your change will remove support for the 1.4 branch once we create it. Was that intentional?

```suggestion
        "/^release-1\\..*/",
```